### PR TITLE
[openui5] Update the definition files for OpenUI5 1.100

### PR DESCRIPTION
**Pull request by akudev**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).  → as detailed in the [previous pull request for 1.90](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/53680), some are unavoidable at this point
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: → no specific URL about the changes, it's just a new release: https://openui5.hana.ondemand.com/1.100.0
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/openui5/sap.ui.table.d.ts [hunk 1]
```diff
@@ -1366,13 +1362,6 @@ declare module "sap/ui/table/Column" {
      * Default value is `false`.
      */
     getAutoResizable(): boolean;
-    /**
```
Comment (codeworrior) [inline types/openui5/sap.ui.table.d.ts:1369]: this method should not disappear in 1.100

**Review by petermuessig — APPROVED** [review]
Approve

File: types/openui5/sap.ui.table.d.ts (hunk 1 above)
Comment (codeworrior) [reply types/openui5/sap.ui.table.d.ts:1369]: Okay, it's in line with OpenUI5 change https://github.com/SAP/openui5/commit/8af8fdb59809da0d7bf538425a91ff771efbd294 .

**Review by codeworrior — APPROVED** [review]
LGTM

**Comment by akudev** [comment]
Ready to merge

**Pull request merged**